### PR TITLE
Wire in the new API method that calculates the total cost and total billable units

### PR DESCRIPTION
**Pull request by minglis**
- [x]  depends on https://github.com/alphagov/notifications-api/pull/983 

Uses the new endpoint to calculate the headline cost/free tier remaining.

### CODE REVIEW

File: app/main/views/dashboard.py [hunk 1]
```diff
@@ -201,6 +201,18 @@ def get_dashboard_totals(statistics):
     return statistics
 
 
+def calculate_fee_tier_usage(usage):
+    # TODO: Don't hardcode these - get em from the API
```
Comment (quis) [inline app/main/views/dashboard.py]: Actually I think it’s fine to have this hard coded in admin. Might be worth pulling it out into config, but not for this PR.

File: tests/app/main/views/test_dashboard.py [hunk 2]
```diff
@@ -684,3 +696,45 @@ def test_should_show_all_jobs_with_valid_statuses(
         'ready to send',
         'sent to dvla'
     })
+
+
+def test_should_show_remaining_fee_tier_count(
```
Comment (imdadahad) [inline tests/app/main/views/test_dashboard.py]: small typo, sorry, `fee_tier` -> `free_tier`

**Comment by imdadahad** [comment]
Other than small typo mentioned looks good 👍

File: tests/app/main/views/test_dashboard.py [hunk 3]
```diff
@@ -684,3 +696,45 @@ def test_should_show_all_jobs_with_valid_statuses(
...
+
+    response = logged_in_client.get(url_for('main.service_dashboard', service_id=SERVICE_ONE_ID))
+
+    assert response.status_code == 200
+    assert '249,900' in response.get_data(as_text=True)
+    assert 'free text messages left' in response.get_data(as_text=True)
+
+
+def test_should_show_cost_if_exceeded_fee_tier_count(
+    logged_in_client,
```
Comment (imdadahad) [inline tests/app/main/views/test_dashboard.py]: one more here: `test_should_show_cost_if_exceeded_fee_tier_count` -> `test_should_show_cost_if_exceeded_free_tier_count`

File: app/main/views/dashboard.py [hunk 4]
```diff
@@ -201,6 +201,18 @@ def get_dashboard_totals(statistics):
     return statistics
 
 
+def calculate_fee_tier_usage(usage):
```
Comment (imdadahad) [inline app/main/views/dashboard.py]: and here `calculate_fee_tier_usage` -> `calculate_free_tier_usage`

**Pull request merged**